src/seaport.ts

```diff
@@ -287,11 +363,10 @@ export class Seaport {
       salt ||
       (domain ? generateRandomSaltWithDomain(domain) : generateRandomSalt());
 
-    const orderParameters: OrderParameters = {
+    const orderComponents: OrderComponents = {
       offerer,
       zone,
-      // TODO: Placeholder
-      zoneHash: formatBytes32String(resolvedCounter.toString()),
+      zoneHash: ethers.constants.HashZero,
```

Comment:
i'm not sure why the zoneHash was being set to the counter here so i changed it to zero hash